perception/detected_object_validation/src/object_lanelet_filter.cpp

```diff
@@ -82,6 +82,13 @@ void ObjectLaneletFilterNode::objectCallback(
     return;
   }
 
+  // if shape is BOUNDING_BOX, make foot print
+  for (auto & object : transformed_objects.objects) {
+    if (object.shape.type == object.shape.BOUNDING_BOX) {
+      object.shape.footprint = setFootprint(object);
+    }
+  }
+
```

Comment:
I think it would be better to define it as a separate structure instead of using the DetectedObject footprint all over again.

Reply:
I cannot get your comment properly, but I tried to fix at d0ef539.
If this change doesn't follow your thought, I ask you to present your thoughts in a detail.

Reply:
That is exactly what I intended. :+1: 
LGTM